apps/src/StudioApp.js

```diff
@@ -333,6 +333,10 @@ StudioApp.prototype.init = function(config) {
   if(config.embed) {
     var resized = false;
     var resize = function() {
+      if (options.app === 'applab') {
```

Comment:
If we go with this approach, please add a comment explaining that (1) the resize logic here is designed to make embedded levels display correctly on mobile, and (2) we can skip this for applab because CSP is not supported on mobile.